engine/test-src/org/pentaho/di/job/entries/copyfiles/JobEntryCopyFilesTest.java

```diff
@@ -74,6 +74,7 @@ public void fileCopied() throws Exception {
 
     verify( entry ).processFileFolder( anyString(), anyString(),
         anyString(), any( Job.class ), any( Result.class ) );
+    verify( entry, atLeast( 1 ) ).preprocessfilefilder( any( String[].class ) );
```

Comment:
I don't see how this one test covers the fact that you are now persisting and restoring job entries with  "EMPTY_DEST_URL-" and "EMPTY_SOURCE_URL-" prefixes.